src/pages/iou/WaypointEditor.js

```diff
@@ -96,6 +104,12 @@ function WaypointEditor({transactionID, route: {params: {iouType = '', waypointI
         Navigation.goBack(ROUTES.getMoneyRequestDistanceTabRoute(iouType));
     };
 
+    const confirmDeleteStopAndHideModal = () => {
```

Comment:
can we rename this to `deleteStopAndHideModal`?

also, is it our a part of the style guideline that we cannot use name like `onConfirm` or something? (I know I'm the one who should know this 😭 )

Reply:
ah yep I remember it's from the checklist... anyway, thank you for the change and detailed explanations 🙇